docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,442 @@
+running ``crate-node unsafe-bootstrap`` on a master-eligible node that belonged
+to the same cluster as this node.
 
-        WARNING: CrateDB MUST be stopped before running this tool.
+.. NOTE: Before you continue with the next step, you need to stop CrateDB.
 
-    You should only run this tool if you have permanently lost all of the
-    master-eligible nodes in this cluster and you cannot restore the cluster
-    from a snapshot, or you have already unsafely bootstrapped a new cluster
-    by running ``crate-node unsafe-bootstrap`` on a master-eligible
-    node that belonged to the same cluster as this node. This tool can cause
-    arbitrary data loss and its use should be your last resort.
+.. WARNING:: This tool can cause arbitrary data loss, and its use should be your
+             last resort.
+
+Run ``crate-node detach-cluster`` and answer ``y`` at the confirmation step.
+
+.. code-block:: console
+
+    sh$ node_3$ ./bin/crate-node detach-cluster
 
     Do you want to proceed?
 
     Confirm [y/N] y
-    Node was successfully detached from the cluster
\ No newline at end of file
+    Node was successfully detached from the cluster.
+
+Recovering data after a disaster
+--------------------------------
+
+Sometimes CrateDB nodes are temporarily stopped to perform maintenance
+activities or because of hardware failure. Depending on your configuration,
+the cluster may be able to remain available even while one or more of the
+cluster's nodes are stopped. After you resolve the temporary condition and
+restart the node, the node rejoins the cluster and continues normally.
+
+Each node stores its data in the data directories defined by the ``path.data``
+setting. Meaning that in a disaster you can restart a node by moving its data
+directories to another host. You can only move data directories to other hosts,
+however, if those data directories can be recovered from the faulty host.
+
+CrateDB requires a response from a majority of the master-eligible nodes
+to be able to elect a master and update the cluster state. A majority of the
+master-eligible nodes responding means that if you have three master-eligible
+nodes, the cluster remains available even if one of the nodes has failed.
+However, if two of the three master-eligible nodes fail the cluster is
+unavailable until at least one of the nodes restarts.
+
+Under certain circumstances, it is not possible to restart enough nodes to
+restore the cluster’s availability. In such a disaster, we recommend you
+build a new cluster from a recent snapshot and reimport any data that was
+ingested since that snapshot was taken.
+
+If it is not possible to recover the cluster from a recent snapshot, it may be
+possible to use the CLI tool to construct a new cluster that contains some of
+the data from the failed cluster.
+
+This part of the CrateDB documentation is based on the `Elasticsearch
+Reference [7.2]. <https://www.elastic.co/guide/en/elasticsearch/reference/current/node-tool.html>`_
```

Comment:
```suggestion
Reference. <https://www.elastic.co/guide/en/elasticsearch/reference/current/node-tool.html>`_
```

also, pull this link out and put it at the end of the document. see other rst files for examples